expenses-manager-frontend/src/main/frontend/src/app/reports/reports.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="datepicker">
+    <app-datepicker [(startDate)]="startDate" [(endDate)]="endDate" (updateRequest)="update()"></app-datepicker>
```

Comment:
De asemenea cand deschid calendarul mi se deschide in partea stanga sus. Ar trebui sa pastram o consistenta, si sa arate la fel ca in expenses list. 
Aceeasi problema si la home, doar ca la home mi se deschide un calendar foarte mare..

Reply:
Am rezolvat această problemă într-un alt PR la care lucrez